### ISSUE TITLE
Phantom traces on the website

### ISSUE THREAD

**Issue by Zverik** [openstreetview/openstreetview.org#35]
See here: http://openstreetview.org/map/@57.96209500840829,27.600488662719727,13z
I didn't drive to Varska, and you cannot click on any point of that segment.

**Comment by james2432** [comment]
Possible duplicate of #26

**Comment by ToeBee** [comment]
I don't think it is a duplicate. I might be wrong but I think you are doing coverage based on OSM ways. If there is a single picture that gets associated with a way, the entire way shows up on the coverage map. Or possibly segments between connected ways? So in this case, [way 43767638](https://www.openstreetmap.org/way/43767638) gets marked as "covered" even though only a couple hundred meters are actually covered.

I see this at intersections a lot where a picture or two get snapped to a way that I didn't actually travel on but that way gets marked as covered anyway.

**Comment by Zverik** [comment]
#1 looks like this. I agree this might be related to using osm ways.

**Comment by ToeBee** [comment]
Here's a pretty obvious example too: 
http://openstreetview.com/map/@40.04430616985666,-96.35640621185303,14z

There are 1 or 2 pictures that could be considered as actually being of the road to the east but I certainly did not travel down that road.

**Comment by alexandrui-telenav** [comment]
It's not a duplicate, as @Zverik presumed it's related to matching the raw positions to OSM ways. 
The matching is quite recently implemented and apparently there are some things to adjust that we didn't catch. Great that you reported this, added this to the backlog.

**Comment by daviewales** [comment]
I think I'm getting this too:
https://openstreetcam.org/map/@-34.228884173379875,150.555557012558,17z

If you load the same OpenStreetCam data on the OpenStreetMap website, it follows the road perfectly:
https://www.openstreetmap.org/edit#map=17/-34.23024/150.55618

(Don't forget to add the OpenStreetCam data from the right sidebar.)
